demos/src/demo.scss

```diff
@@ -42,3 +42,18 @@ body {
 	background: oColorsGetPaletteColor('grey-tint5');
 }
 
+// Set the backgrounds for hover states as actual hex values using mix()
+// because Pa11y isn't liking the transparent backgrounds that are set in our colours
```

Comment:
It's not really an issue in Pa11y, it's [HTML_CodeSniffer](https://github.com/squizlabs/HTML_CodeSniffer) which actually tests the page. The only issue I found was this: https://github.com/squizlabs/HTML_CodeSniffer/pull/96